### PR TITLE
timeline graphic

### PR DESCRIPTION
**Pull request by foxtree**
This is on top of #59 

1st pass:
image

https://user-images.githubusercontent.com/3383704/110256141-43273700-7f65-11eb-95bc-c19c8e1c1170.mov

TODO:
indicate bars beyond 1 and 2 standard deviations (z score?)

### CODE REVIEW

**Review by gilhsu — COMMENTED** [review]
I see this is marked for PR but may be still in progress? The timeline graphic does not appear to be responding to the resourceType filters.

File: src/components/Timeline/index.js [hunk 1]
```diff
@@ -2,22 +2,25 @@ import React from 'react';
...
 
 const styles = StyleSheet.create({
   root: {
     width: '100%',
-    flexDirection: 'row',
-    height: hp('20%'),
+    flexDirection: 'column',
+    // minHeight: 120,
+    // height: hp('20%'),
     backgroundColor: 'lightgray',
```
Comment (gilhsu) [inline src/components/Timeline/index.js]: We should get rid of the placeholder backgroundColor.

**Comment by foxtree** [comment]
> I see this is marked for PR but may be still in progress? The timeline graphic does not appear to be responding to the resourceType filters.

It was still draft-y, until a couple hours ago.

resourceType filter should now work.

**Review by gilhsu — APPROVED** [review]
Very nice

**Pull request merged**